server/base/src/main/java/org/apache/accumulo/server/manager/state/MetaDataStateStore.java

```diff
@@ -60,9 +62,24 @@ public ClosableIterator<TabletLocationState> iterator() {
   public void setLocations(Collection<Assignment> assignments) throws DistributedStoreException {
     try (var tabletsMutator = ample.mutateTablets()) {
       for (Assignment assignment : assignments) {
-        tabletsMutator.mutateTablet(assignment.tablet)
-            .putLocation(assignment.server, LocationType.CURRENT)
-            .deleteLocation(assignment.server, LocationType.FUTURE).deleteSuspension().mutate();
+        TabletMutator mutation = tabletsMutator.mutateTablet(assignment.tablet);
```

Comment:
I would just change this variable name, so it's not confused for a `Mutation` type object, and so it's consistent with the other places we do this.

```suggestion
        TabletMutator tabletMutator = tabletsMutator.mutateTablet(assignment.tablet);
```

Reply:
noted.  I will do that.